src/app/components/music-controls/music-controls.component.html

```diff
@@ -0,0 +1,43 @@
+<div class="controls-wrapper">
+    <div class="play-controls">
+        <button class="transparent_button" (click)="playPreviousTrack()">
+            <i class="material-icons" id="arrow">arrow_left</i>
+        </button>
+        <button class="transparent_button" (click)="playNextTrack()" [disabled]="isTracksListEnd">
+            <i class="material-icons" id="arrow" [style.opacity]="isTracksListEnd ? '0.6' : '1'">arrow_right</i>
+        </button>
+    </div>
+    <button class="transparent_button" (click)="autorenew()">
+        <i class="material-icons" [style.opacity]="isAutorenewed ? '1' : '0.6'">autorenew</i>
+    </button>
+    <button class="transparent_button" (click)="playPause()">
+        <i class="material-icons" id="play" *ngIf="state?.playing">pause_circle_outline</i>
```

Comment:
these ids are useless. There is no need to use a structural directive to change only text inside. You can try something like:
`<i class="material-icons">{{state?.playing ? 'pause_circle_outline' : 'play_circle_outline'}}</i>`
Try to reduce template manipulation